src/PhysicsEngine.cpp

```diff
@@ -185,3 +184,257 @@ void ForwardEuler::march_step(double tStamp, double tStep) {
     rocket_.set_t_net(t_net_if);
     rocket_.set_q_ornt(q_ornt);
 }
+
+/**
+ * @brief Averages forces and moments for a more accurate euler step
+ * 
+ * @param tStamp 
+ * @param tStep 
+ */
```

Comment:
This docstring needs some work.

I think the `@brief` is not accurate. I'd suggest: "Performs a fourth-order Runge Kutta method to advance the physics forward one timestep."

After `@brief` you can add more information in the form of paragraphs. Since this is a math heavy function, you should put some sources of where you got the math from. Check out the function I wrote in [[AV-346] PR](https://github.com/ISSUIUC/ISS_SILSIM/pull/19) as an example

Also please briefly explain what the `@param`s are in one line